src/components/MultiSelect.js

```diff
@@ -18,43 +33,71 @@ import {
   Typography,
 } from "@components";
 
+const Typography2 = styled.Text`
```

Comment:
We can name this as `Typography` itself and remove the same from the above import and use it in the file.